src/db/migrations/26-add-crdt-messages.ts

```diff
@@ -0,0 +1,47 @@
+import { Knex } from 'knex';
+
+import { CrdtOpetation } from '../../types/message';
+import { Table } from '../db';
+import { tableNameToViewName, updateViews } from '../migration-helpers';
+
+export const up = async (knex: Knex) => {
+  await knex.schema.createTable(Table.seeds, table => {
+    table.dateTime('timestamp');
+    table.enu('table', Object.values(Table));
```

Comment:
q - should we use an enum for this or just a string?
i guess enum is good in that it gives us a kind of type safety, but means we need to remember to update this enum on all 3 tables whenever we add a new table to the system, similar to how we now need to call updateViews() every migration? i guess that's fine still though - i'm assuming there's no way for the enum to be a type that automatically/dynamically tracks table names?

Reply:
I think the way to do it dynamically would be to create a table and then use FK references here instead of an enum. That's nice because of the type safety, and we also have a dynamic list of tables that can be used in messages. 

The same table could be referenced here https://github.com/spinamp/spindexer/blob/7c9e3d44e1bc7c8e15ea590bf8af1edade1a6881/src/index.ts#L42-L49

Reply:
hmm ye though we'd still need to make sure that new tables are added to that list of tables lol. i suppose that can be automated but yeh does seem like overkill for now? maybe fine to leave as enum as is?